src/lib.rs

```diff
@@ -2546,7 +2550,7 @@ impl Build {
             .as_ref()
             .map(|s| s.trim_right_matches('-').to_owned());
         cross_compile.or(match &target[..] {
-            "aarch64-pc-windows-gnu" => Some("aarch64-w64-mingw32"),
+            "aarch64-pc-windows-gnullvm" => Some("aarch64-w64-mingw32"),
```

Comment:
Does this mean to remove the `aarch64-pc-windows-gnu` target?

Reply:
Yes, there is no AArch64 support for Windows in both Binutils and GCC. If you insist I can restore it but it's dead code.

Reply:
Can you add a comment to this effect, then?

Reply:
Comment in commit message/PR description or something else?
There was no `aarch64-pc-windows-gnu` target ever available in Rust.

Reply:
Before that arm, something like this maybe?
```rs
// Note: there is no `aarch64-pc-windows-gnu` target, only `-gnullvm`
```